pkgs/tools/package-management/nix/common.nix

```diff
@@ -176,6 +177,7 @@ self = stdenv.mkDerivation {
   installFlags = [ "sysconfdir=$(out)/etc" ];
 
   doInstallCheck = true;
+  installCheckTarget = if atLeast210 then "installcheck" else null;
```

Comment:
```suggestion
  installCheckTarget = lib.optionalString atLeast210 "installcheck";
```
nit

Reply:
I was trying to reduce the number of rebuild, but given that a lot of old versions were removed, this wouldn't have made a deep dent.